app/models/service_template.rb

```diff
@@ -411,6 +412,10 @@ def add_resource(rsc, options = {})
     set_service_type
   end
 
+  def archived
+    archived?
```

Comment:
why is this needed? can virtual columns not have a `?`

Reply:
Oh I guess it wouldn't work well with the URI then, still new to me so bear with me.
You should add one for `active` also then if we want to use that in the future

Reply:
`?` have a special meaning in the API context - so I was just making sure to keep things simple in the context of the API
Although I have verified that this does work -
```
virtual_column   :archived?,                     :type => :boolean
```

LMK if you want me to change it.

Reply:
The thing is these are defined in the archived mixin so we would need to add these to every model that includes the mixin, might be better to either use the methods that the mixin provides or add aliases and virtual-columns to the mixin

Reply:
Ok, let's go with `archived?`, since the API can handle it